docs/kyma/docs/04-05-use-your-own-image.md

```diff
@@ -0,0 +1,85 @@
+---
```

Comment:
I think I missing some basing understanding of the installation from custom image, or there is just not enough intro. 
This docs first focuses on building image, and then creating of own my-kyma.yaml

We need more explanation on why I would even consider that. I assume that use case is that I build a custom Installer image from my modified sources of Kyme. I imagine a use case that someone wants to install latest `master` kyma on a cluster and to do that he needs to follow these steps. Please confirm and update

Reply:
another use case is what I've learned from `05-01-overview.md` and makes a lot of sense, that you want to create installer with Kyma + own custom helm charts

Reply:
ok, done :)